analyzers/tests/SonarAnalyzer.UnitTest/Rules/AvoidExcessiveClassCouplingTest.cs

```diff
@@ -27,25 +27,27 @@ namespace SonarAnalyzer.UnitTest.Rules
     [TestClass]
     public class AvoidExcessiveClassCouplingTest
     {
+        private readonly VerifierBuilder builderWithThreshold0 = new VerifierBuilder().AddAnalyzer(() => new AvoidExcessiveClassCoupling { Threshold = 0 });
```

Comment:
Could be like this for simplicity
```suggestion
        private readonly VerifierBuilder withThreshold0 = new VerifierBuilder().AddAnalyzer(() => new AvoidExcessiveClassCoupling { Threshold = 0 });
```